annot8-components-easyocr/src/main/java/io/annot8/components/easyocr/processors/LocalEasyOCR.java

```diff
@@ -0,0 +1,256 @@
+/* Annot8 (annot8.io) - Licensed under Apache-2.0. */
+package io.annot8.components.easyocr.processors;
+
+import io.annot8.api.capabilities.Capabilities;
+import io.annot8.api.components.annotations.ComponentDescription;
+import io.annot8.api.components.annotations.ComponentName;
+import io.annot8.api.context.Context;
+import io.annot8.api.exceptions.BadConfigurationException;
+import io.annot8.api.settings.Description;
+import io.annot8.common.components.AbstractProcessorDescriptor;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.function.Consumer;
+
+@ComponentName("Local EasyOCR")
+@ComponentDescription("Perform OCR using the locally installed EasyOCR library")
+public class LocalEasyOCR
+    extends AbstractProcessorDescriptor<LocalEasyOCR.Processor, LocalEasyOCR.Settings> {
+
+  // @formatter:off
+  private static final String OCR_PY =
+      String.join(
+          System.getProperty("line.separator"),
+          "from fastapi import FastAPI, Response, File",
+          "from typing import Optional",
+          "from pydantic import BaseModel",
+          "import easyocr",
+          "",
+          "class Config(BaseModel):",
+          "    langs: Optional[str] = \"en\"",
+          "    download: Optional[bool] = False",
+          "    gpu: Optional[bool] = False",
+          "",
+          "app = FastAPI()",
+          "",
+          "",
+          "reader = easyocr.Reader([\"en\"], download_enabled=False, gpu=False)",
+          "",
+          "",
+          "@app.post(\"/init\")",
+          "def init(config: Config):",
+          "    langs = config.langs.split(\",\")",
+          "    global reader",
+          "    reader = easyocr.Reader(langs, download_enabled=config.download, gpu=config.gpu)",
+          "    ",
+          "@app.post(\"/ocr\")",
+          "def index(file: bytes = File(...)):",
+          "    results = reader.readtext(file, detail=0, paragraph=True, y_ths = -0.01, x_ths = 10.0)",
+          "    return Response(content=\"\\n\\n\".join(results), media_type=\"text/plain\")");
+  // @formatter:on
+
```

Comment:
I wonder if it's worth moving this to a separate Python file (i.e. in a resource), so that we can use syntax highlighting, code completion, etc on any changes/future development?

Reply:
It is in the server folder as a .py file. I'd like to avoid the duplication and I tried the resources route but it hit a couple of issues so was easier to cut and paste from there.

In dev, it writes the file out so is easy to check. 
I'll add a comment to direct.